apis_core/generic/views.py

```diff
@@ -213,7 +225,8 @@ class Autocomplete(
 
     permission_action_required = "view"
     template_name_suffix = "_autocomplete_result"
-    create_field = "thisisnotimportant"  # because we are using create_object_from_uri
```

Comment:
Please undo this change

Reply:
but thats just formatting (probably done by ruff?)
or is it that you'll prefer formatting in a different PR/commit?

Reply:
never mind, given that its just formatting removed it